### PR TITLE
PHOENIX-7261 Align mockito version with Hadoop and HBase in QueryServer

### PR DESCRIPTION
**Pull request by NihalJain**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
 
         <!-- Test Dependency versions -->
         <testng.version>6.10</testng.version>
-        <mockito-all.version>1.8.5</mockito-all.version>
+        <mockito-all.version>4.11.0</mockito-all.version>
```
Comment (stoty) [inline pom.xml:98]: We are not using mockito-all, better rename this to mockito.version

**Review by stoty — CHANGES_REQUESTED** [review]
Let's clean up the property name, too.

File: pom.xml (hunk 1 above)
Comment (NihalJain) [reply pom.xml:98]: sure @stoty let me update

Comment (NihalJain) [reply pom.xml]: done

**Review by stoty — APPROVED** [review]
+1 lgtm

**Pull request merged**